code/game/objects/items/theft_tools.dm

```diff
@@ -263,3 +263,24 @@
 	playsound(src, 'sound/effects/supermatter.ogg', 50, TRUE)
 	QDEL_NULL(sliver)
 	update_icon()
+
+//Don't know where else to put fancy syndicate tools
```

Comment:
"tgstation/code/game/objects/items/tools/" is a folder you could use to store the new tools in their respective files. This file is for the tools related to stealing the nuke core and Supermatter piece.

Reply:
Thanks, I did not know if that was a good spot either.